src/svelte/src/utilities/elementKeypressEvents.ts

```diff
@@ -0,0 +1,64 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+type ElementKeypressEventKey = 'Enter' | 'Delete'
+const MappableKeys: { [k in ElementKeypressEventKey]: true } = {
+  Enter: true,
+  Delete: true,
+}
```

Comment:
So we have this elementKeypressEvents class that really isn't generic as it's specific to Enter and Delete.  Can it be made generic?

Reply:
@scholarsmate I was going to make it completely generic but left it as `Enter` and `Delete` only as I wasn't sure if we wanted to ability to make _all_ keys mappable. 

I guess it doesn't matter much though, outside of the conditional in `dataEditor.svelte` where it checks if any key that triggers the event is a mappable key.

Reply:
Yeah, it feels like overkill right now, but I _could_ see it expanding to becoming something more generic that can be used to handle all of our keybinding.  We may also want to allow the user to be able to configure their keybindings at some point down the road like many other editors and products allow.